inc/functions_post.php

```diff
@@ -1021,3 +1021,28 @@ function get_post_attachments($id, &$post)
 		}
 	}
 }
+
+/**
+ * Returns bytes count from human readable string
+ * Used to parse ini_get human-readable values to int
+ *
+ * @param string $val Human-readable value
+ */
+function return_bytes($val) {
+    $val = trim($val);
+    if ($val == "")
+    	return 0;
+
+    $last = strtolower($val[strlen($val)-1]);
+    switch($last) {
+        // The 'G' modifier is available since PHP 5.1.0
+        case 'g':
+            $val *= 1024;
+        case 'm':
+            $val *= 1024;
+        case 'k':
+            $val *= 1024;
```

Comment:
```
        case 'g':
        case 'm':
        case 'k':
            $val *= 1024;
        default:
            $val = 0;
```
Better?

Reply:
No, as 'g' is supposed to multiply by 2^24, 'm' by 2^16 and 'k; by 2^8